app/models/lawyer.rb

```diff
@@ -7,15 +7,15 @@ class Lawyer < ApplicationRecord
   validates_associated :reservations
   validates :name, presence: true, length: { maximum: 50 }
   validates :phone, presence: true, length: { maximum: 15 }, numericality: { only_integer: true }
-  validates :rates, comparison: { greater_than: 0 }, format: { with: /\A\d+(?:\.\d{0,2})?\z/,
```

Comment:
For the phone number, We were supposed to add validation so that it accommodates the special symbols like "+". I think it would be better if we have the type as string instead of "only integer".

Reply:
Thank you @richardoppiyo , I have updated the phone column's type, and added regex to phone validation.